components/dashboard/ExportTransactionsCSVModal.tsx

```diff
@@ -113,6 +120,18 @@ const makeUrl = ({ account, isHostReport, queryFilter, flattenTaxesAndPaymentPro
     url.searchParams.set('expenseId', queryFilter.values.expenseId);
   }
 
+  if (queryFilter.values.merchantId) {
+    url.searchParams.set('merchantId', queryFilter.values.merchantId);
```

Comment:
On the API side, we also support a list of strings as merchantId.

Reply:
Yep, keeping this as a single value in the filters here to make the input component just need to care about a single value. Since the REST API expects a string delimited with `,` this should work regardless.